media/json-jackson/src/main/resources/META-INF/NOTICE.markdown

```diff
@@ -1,4 +1,20 @@
-# Notice for Jersey Json Jackson module 
+#
+# Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
this is kinda .md file (only with .markdown extension), shall be excluded from Copyright check.

Reply:
Done, it is also added in the .markdown in the jersey/etc/config/copyright-exclude